tests/unit/enums_py/StatusCode_test.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+from iomirea_rpc.enums import StatusCode
+
+class enums_StatusCode_TestCase(unittest.TestCase):
+    """
+    Testing the StatusCode Class
+    """
+
+    def test_enums_variables_values(self):
+        """
+        Should ensure that variables are set to their expected values
+        """
+
+        all_enumarations_values = [
+            StatusCode.SUCCESS.value,
+            StatusCode.BAD_FORMAT.value,
```

Comment:
Can `all_enumarations_values` be moved outside to prevent code duplication?

Reply:
I will clean my code.